scrape/scrape.go

```diff
@@ -378,9 +378,24 @@ func (s *targetScraper) scrape(ctx context.Context, w io.Writer) error {
 		return fmt.Errorf("server returned HTTP status %s", resp.Status)
 	}
 
-	_, err = profile.Parse(io.TeeReader(resp.Body, w))
-	if err != nil {
-		return errors.Wrap(err, "failed to parse target's pprof profile")
+	switch profileType {
+	case ProfileTraceType:
+		// This is needed as trace.Parse fails with obscure error
+		// when you pass resp.Body
```

Comment:
do we know why?

Reply:
I didn't dig deeply but the error is:

```
failed to parse target's trace profile: failed to read event 3 argument at offset 0 (failed to read trace at offset 13621: read 1, error EOF)
```

Reply:
Odd, let’s try to come back to it. Could you open an issue for it?

Reply:
https://github.com/conprof/conprof/issues/60